cabal.haskell-ci

```diff
@@ -0,0 +1,7 @@
+apt:
```

Comment:
This config is used by haskell-ci. It lists additional ubuntu packages that are installed in CI image.
I have no idea what these packages are. I just followed compiler errors, googled around and added them one by one to make everything compile :smiley: